packages/components/src/informative/UserDisplayItem/UserDisplayItem.js

```diff
@@ -84,7 +84,10 @@ const UserDisplayItem = (properties) => {
 
   const role = useMemo(() => (!isEmpty(center) ? `${rol} · ${center}` : rol), [rol, center]);
   const fullName = useMemo(
-    () => (['rol', 'inline'].includes(variant) ? `${surnames || ''} ${name}` : name),
+    () =>
+      ['rol', 'inline'].includes(variant)
+        ? `${surnames || ''}${!surnames ? '' : ', '} ${name}`
```

Comment:
If the space is already added in the comma, then no space required before name

```suggestion
        ? `${surnames || ''}${!surnames ? '' : ', '}${name}`
```

Reply:
ok. changed.